src/elexmodel/handlers/data/VersionedData.py

```diff
@@ -222,9 +231,8 @@ def get_versioned_predictions(self, filepath=None):
             return pd.read_csv(filepath)
 
         if self.election_id.startswith("2020-11-03_USA_G"):
-            path = "elex-models-prod/2020-general/prediction/pres/current.csv"
```

Comment:
why has this been removed?

Reply:
Once the exception is raised, `path` is unused.  We don't need it 🤷🏻‍♀️